ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-installer-rehearse.yaml

```diff
@@ -39,6 +39,11 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: installer-rehearse-azure-ipi-sharednetwork
+  interval: 960h
```

Comment:
For easier validation, actually you do not need to add a new test confi here, you can directly change the line 41 to use `cucushift-installer-rehearse-azure-ipi-sharednetwork-workflow.yaml` workflow. After that, you even do not need to run `make job`